baseplate/lib/metrics.py

```diff
@@ -186,6 +199,17 @@ def batch(self) -> "Batch":
         """
         return Batch(self.transport, self.namespace)
 
+    def tagged_batch(self) -> "Batch":
```

Comment:
I don't think we need this now, right? Since we're recommending modern services don't put a namespace at all in favor of telegraf doing it for them?

Reply:
I added this just so we could get a Batch with an empty namespace, since it's something attached automatically in batch() without being passed in

Reply:
Is there another way I could do this? I didn't want to redo the batch() function since that might hurt backwards compatibility

Reply:
I think we could just ignore namespaces altogether, right? and make sure we throw an error if they used tags + a configured namespace?

Reply:
* set a rule that says namespace must be blank in config